gmusicapi/clients/musicmanager.py

```diff
@@ -424,7 +420,7 @@ def upload(self, filepaths, enable_matching=False,
                                       path, sample_request, track,
                                       self.uploader_id, bogus_sample)
 
-            except (IOError, ValueError) as e:
+            except (OSError, ValueError) as e:
```

Comment:
There's a few mentions of IOError in docstrings that we should probably switch too.

Reply:
Fixed!